projects/laji/src/app/+information/information.module.ts

```diff
@@ -4,9 +4,9 @@ import { SharedModule } from '../shared/shared.module';
 import { routing } from './information.routes';
 import { InformationComponent } from './information.component';
 import { LajiUiModule } from 'projects/laji-ui/src/public-api';
-
+import { TechnicalNewsModule } from '../shared-modules/technical-news/technical-news.module';
```

Comment:
Not a big issue but it looks a little weird that there is no space between the lines `import ...` and `@NgModule ...`